### PR TITLE
adding fertilizer EFs

### PR DESCRIPTION
**Pull request by AnastasiaLobanova**
**Is this a new addition or an update to an existing factor?**

- [x] New
- [ ] Update

**What category or categories does it affect?**

Arable farming

**What?**

adding fertilizer EFs

**Why?**

new EFs ingress

### CODE REVIEW

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -21651,3 +21651,33 @@ dc0ba7ab-9069-4293-8818-7ed78e3675a0,Energy,Fuel,fuel_type_natural_gas-fuel_use_
...
+4fa7a6e4-536d-4b0e-a3a5-8f5c3068dfd1,Agriculture/Hunting/Forestry/Fishing,Arable Farming,arable_farming-fertilizer_urea_wet_climate_indirect,Indirect emissions from urea fertilizer application in wet climate,kg,not-supplied,not-supplied,0.0,0.0,0.0033,not-supplied,not-supplied,,,use_phase,IPCC,2019 Refinement to the 2006 IPCC Guidelines for National Greenhouse Gas Inventories,2019,2019,,GLOBAL,,Climatiq,2023/02/10,"Indirect emissions from N applications. We have included volatilisation only assuming that leaching and runoff are not applicable for horticulture in private gardens. From IPCC ""2019 Refinement to the 2006 IPCC Guidelines for National Greenhouse Gas Inventories"". Chapter 11. Wet / dry climate from: Menegat S. Ledo A. & Tirado R. Greenhouse gas emissions from global production and use of nitrogen synthetic fertilisers in agriculture. Sci Rep 12 14490 (2022). ",https://www.ipcc.ch/report/2019-refinement-to-the-2006-ipcc-guidelines-for-national-greenhouse-gas-inventories/
```
Comment (TimCroker) [inline OpenEmissionFactorsDB.csv:21654]: arable_farming should be replaced with crop_production throughout

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -21651,3 +21651,33 @@ dc0ba7ab-9069-4293-8818-7ed78e3675a0,Energy,Fuel,fuel_type_natural_gas-fuel_use_
...
+5392bacb-84fb-4d34-abfd-94b00f99341a,Agriculture/Hunting/Forestry/Fishing,Arable Farming,arable_farming-fertilizer_nitrate_based_wet_climate_indirect,Indirect emissions from nitrate based fertilizer application in wet climate,kg,not-supplied,not-supplied,0.0,0.0,0.00022,not-supplied,not-supplied,,,use_phase,IPCC,2019 Refinement to the 2006 IPCC Guidelines for National Greenhouse Gas Inventories,2019,2019,,GLOBAL,,Climatiq,2023/02/10,"Indirect emissions from N applications. We have included volatilisation only assuming that leaching and runoff are not applicable for horticulture in private gardens. From IPCC ""2019 Refinement to the 2006 IPCC Guidelines for National Greenhouse Gas Inventories"". Chapter 11. Wet / dry climate from: Menegat S. Ledo A. & Tirado R. Greenhouse gas emissions from global production and use of nitrogen synthetic fertilisers in agriculture. Sci Rep 12 14490 (2022). ",https://www.ipcc.ch/report/2019-refinement-to-the-2006-ipcc-guidelines-for-national-greenhouse-gas-inventories/
```
Comment (TimCroker) [inline OpenEmissionFactorsDB.csv:21656]: Change all instances of arable_farming to crop_production

**Review by TimCroker — CHANGES_REQUESTED** [review]

**Review by TimCroker — DISMISSED** [review]

File: OpenEmissionFactorsDB.csv [hunk 3]
```diff
@@ -21651,3 +21651,33 @@ dc0ba7ab-9069-4293-8818-7ed78e3675a0,Energy,Fuel,fuel_type_natural_gas-fuel_use_
...
+44786e2f-2389-4ce4-9554-007379b3d394,Agriculture/Hunting/Forestry/Fishing,Arable Farming,arable_farming-fertilizer_urea_wet_climate_indirect,Indirect emissions from urea fertilizer application in wet climate,kg,not-supplied,not-supplied,0.0,0.0,0.00745,not-supplied,not-supplied,,,use_phase,IPCC,2019 Refinement to the 2006 IPCC Guidelines for National Greenhouse Gas Inventories,2019,2019,,GLOBAL,,Climatiq,2023/02/10,"Indirect emissions from N applications. Includes volatilization, leaching and runoff. From IPCC ""2019 Refinement to the 2006 IPCC Guidelines for National Greenhouse Gas Inventories"". Chapter 11 Equation 11.10. Wet / dry climate country match from: Menegat S. Ledo A. & Tirado R. Greenhouse gas emissions from global production and use of nitrogen synthetic fertilisers in agriculture. Sci Rep 12 14490 (2022). ",https://www.ipcc.ch/report/2019-refinement-to-the-2006-ipcc-guidelines-for-national-greenhouse-gas-inventories/
```
Comment (TimCroker) [inline OpenEmissionFactorsDB.csv:21654]: Slight typo: revise, for all INDIRECT EFs:
_From:_
From IPCC ""2019 Refinement to the 2006 IPCC Guidelines for National Greenhouse Gas Inventories"". Chapter 11 Equation 11.10.
_To_
From IPCC ""2019 Refinement to the 2006 IPCC Guidelines for National Greenhouse Gas Inventories"". Chapter 11 Equations 11.10 and 11.11.

File: OpenEmissionFactorsDB.csv [hunk 4]
```diff
@@ -21651,3 +21651,33 @@ dc0ba7ab-9069-4293-8818-7ed78e3675a0,Energy,Fuel,fuel_type_natural_gas-fuel_use_
...
+2414a1d1-2d07-492b-adbe-6c59fddcdc59,Agriculture/Hunting/Forestry/Fishing,Arable Farming,arable_farming-fertilizer_ammonium_based_wet_climate_indirect,Indirect emissions from ammonium based fertilizer application in wet climate,kg,not-supplied,not-supplied,0.0,0.0,0.00591,not-supplied,not-supplied,,,use_phase,IPCC,2019 Refinement to the 2006 IPCC Guidelines for National Greenhouse Gas Inventories,2019,2019,,GLOBAL,,Climatiq,2023/02/10,"Indirect emissions from N applications. Includes volatilization, leaching and runoff. From IPCC ""2019 Refinement to the 2006 IPCC Guidelines for National Greenhouse Gas Inventories"". Chapter 11 Equations 11.10 and Equation 11.1. Wet / dry climate country match from: Menegat S. Ledo A. & Tirado R. Greenhouse gas emissions from global production and use of nitrogen synthetic fertilisers in agriculture. Sci Rep 12 14490 (2022). ",https://www.ipcc.ch/report/2019-refinement-to-the-2006-ipcc-guidelines-for-national-greenhouse-gas-inventories/
```
Comment (TimCroker) [inline OpenEmissionFactorsDB.csv:21655]: Slight typo: revise, for all INDIRECT EFs:
_From:_
From IPCC ""2019 Refinement to the 2006 IPCC Guidelines for National Greenhouse Gas Inventories"". Chapter 11 Equations 11.10 and Equation 11.1
_To_
From IPCC ""2019 Refinement to the 2006 IPCC Guidelines for National Greenhouse Gas Inventories"". Chapter 11 Equations 11.10 and 11.11.

**Review by TimCroker — CHANGES_REQUESTED** [review]

**Review by TimCroker — APPROVED** [review]

**Review by gpantelidou — APPROVED** [review]

**Pull request merged**